pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -41,6 +41,7 @@ const (
 	UnauthenticatedGroup    = "system:unauthenticated"
 	ClusterAdminGroup       = "system:cluster-admins"
 	ClusterReaderGroup      = "system:cluster-readers"
+	StorageAdminGroup       = "system:storage-admin"
```

Comment:
I'm not sure I would auto-bind the role to a non-existent group. I would start by just defining the role